test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ConditionalAttributes_Runtime.codegen.cs

```diff
@@ -62,13 +62,14 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_Conditiona
             WriteLiteral(" />\r\n    <p");
             BeginWriteAttribute("class", " class=\"", 248, "\"", 281, 1);
             WriteAttributeValue("", 256, new Microsoft.AspNetCore.Mvc.Razor.HelperResult(async(__razor_attribute_value_writer) => {
+                PushWriter(__razor_attribute_value_writer);
 #line 10 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ConditionalAttributes.cshtml"
                if(cls != null) { 
 
 #line default
 #line hidden
 #line 10 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ConditionalAttributes.cshtml"
-WriteTo(__razor_attribute_value_writer, cls);
+                            Write(cls);
```

Comment:
See, this is awesomesauce. Since the prefix is no longer overly long, we can have code gen line up with source as its supposed to be. Debugging win